packages/suite/src/components/suite/landing/ConnectDevice/index.tsx

```diff
@@ -185,7 +185,7 @@ class ConnectDevice extends PureComponent<Props> {
                                     values={{
                                         link: (
                                             <StyledLink to="/bridge" isGreen>
-                                                Trezor Bridge
+                                                <FormattedMessage {...l10nMessages.TR_BRIDGE} />
```

Comment:
unnecessary: "Trezor Birdge" is a application name and it will not be transalated